app/perftools/management/commands/create_jtbd_cache.py

```diff
@@ -261,6 +289,7 @@ def handle(self, *args, **options):
         operations.append(create_jtbd_learn_cache)
         operations.append(create_jtbd_connect_cache)
         operations.append(create_jtbd_fund_cache)
+        operations.append(create_about_cache)
```

Comment:
I think in this way all the marketing landing pages caches are together.  
"create_page_cache" is a big task with a lot of work loads to generate data so I want to avoid adding anything there.